extensions/cornerstone/src/init.tsx

```diff
@@ -426,10 +426,10 @@ function _showCPURenderingModal(uiModalService, hangingProtocolService) {
     }
   };
 
-  const { unsubscribe } = hangingProtocolService.subscribe(
-    hangingProtocolService.EVENTS.HANGING_PROTOCOL_APPLIED_FOR_VIEWPORT,
-    ({ progress }) => {
-      const done = callback(progress);
+  const { unsubscribe } = HangingProtocolService.subscribe(
+    hangingProtocolService.EVENTS.PROTOCOL_CHANGED,
+    () => {
+      const done = callback(100);
```

Comment:
why don't we just say `uiModalService.show` instead of defining a callback and .....

Reply:
I have the same question as @sedghi.